### DISCUSSION TITLE
Problem with json parsing for company announcements

### DISCUSSION THREAD

**Discussion by ranch-water21** [JohnVonNeumann/pyasx#24] [General]
Been using this for a while and its been great. Seems like there might be some form of a Captcha or Incapsula blocking access to the ASX market announcements - is anyone else getting this error. Just started in the last week.

**Comment by JohnVonNeumann** [comment]
Hey mate, I'm glad that you've been getting some good use out of it. To be honest with you, I'd forgotten about this project as I've moved onto something else so I can't really provide much in the way of assistance. I will actually be releasing the pypi package name as I have no interest in supporting this anymore. Feel free to yack the code and do with it as you please. Thanks for the message anyway, always nice to hear people getting some use out of something I've helped contribute to. Good luck with your coding.
